### PR TITLE
chore: update heroku deployment docs

### LINKED ISSUE #4754: Update heroku deployment instructions
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-server

**Current behavior:**
The docs for deployment on Heroku are outdated and a few commands don't work. For eg. the last command ```heroku run migrate``` in the guide to create tables doesn't work. Infact there is no migrate command. 

**Expected behavior:**
The docs should be updated.

### PR DESCRIPTION
**Pull request by srv-twry**
Fixes #4754 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Updated Heroku deployment docs.

#### Changes proposed in this pull request:
- Removed redundant Node JS buildpacks instructions. It even leads to errors like 
```
remote: -----> App not compatible with buildpack: https://codon-buildpacks.s3.amazonaws.com/buildpacks/heroku/nodejs.tgz
remote:        Node.js: package.json not found in application root
```
- Updated the tables creation and database migration instructions.

### CODE REVIEW

File: docs/installation/heroku.md [hunk 1]
```diff
@@ -25,10 +25,13 @@ One-click Heroku deployment is available:
 If they are missing, some problem has occured in the previous steps.
 * Add python build pack to the app
     * ```heroku buildpacks:set heroku/python```
-* Add Node.js build pack to the app
-    * ```heroku buildpacks:add --index 2 heroku/nodejs```
```
Comment (srv-twry) [inline docs/installation/heroku.md]: @niranjan94 I was able to deploy it without using it. I think it's from the version 1 when we had both server and frontend. 
Please correct if wrong.

Comment (poush) [reply docs/installation/heroku.md]: Yes nodeJS build pack is not required since we are no longer using node.

**Comment by srv-twry** [comment]
Deployment link : https://open-event-server-dev-srv-twry.herokuapp.com/

File: docs/installation/heroku.md (hunk 1 above)
Comment (maxlorenz) [reply docs/installation/heroku.md]: And it fails as no `package.json` can be found

Comment (srv-twry) [reply docs/installation/heroku.md]: Yes, exactly.

**Pull request merged**